### PR TITLE
Adding Netperf and DNS from perf-tests repo  

### PR DESCRIPTION
**Pull request by KesavanKing**
Added Network Performance and DNS from perf-tests repo
Modified flags to run clusterloader, Netperf , kube-dns and core-dns
Incorporated using the PR https://github.com/kubernetes/perf-tests/pull/169

### CODE REVIEW

**Comment by spiffxp** [comment]
Context? Ie: Why is this change being proposed. Bonus points if it's a link to a documented discussion ie: issue or mailing list or doc

**Comment by KesavanKing** [comment]
We are intended  to run Performance Tests in  CI pipeline. Currently it invokes only cluster loader. But we like to run network performance and DNS as well. This PR willl help us to run  these perf-tests independently from kubetest.

File: kubetest/main.go [hunk 1]
```diff
@@ -56,69 +56,72 @@ var (
 )
 
 type options struct {
```
Comment (BenTheElder) [inline kubetest/main.go]: unrelated to this PR, but this struct is still fairly horrifying @krzyzacy 😱

File: kubetest/main.go [hunk 2]
```diff
@@ -56,69 +56,72 @@ var (
...
+	kubemarkNodes      	 string // TODO(fejta): switch to int after migration
+	logexporterGCSPath 	 string
+	metadataSources    	 string
+	multiClusters      	 multiClusterDeployment
+	multipleFederations	 bool
+	noAllowDup         	 bool
+	nodeArgs           	 string
+	nodeTestArgs       	 string
+	nodeTests          	 bool
+        perfTestsClusterLoader   bool
```
Comment (KesavanKing) [inline kubetest/main.go]: Included flags to call clusterloader,netperf and dns.
Other lines are modified to have proper alignment

**Comment by krzyzacy** [comment]
nooooooo

you probably should use `--test-cmd` and `--test-cmd-args` instead?

**Comment by krzyzacy** [comment]
flags like `--perf-tests` are legacy and need/will be deprecated soon

/ok-to-test
/assign

**Comment by fejta** [comment]
Please pass tests

**Comment by fejta** [comment]
/approve
/uncc
/assign @krzyzacy 

Sen/Kesavan please assign to me if you need me to do the review instead of Sen. Generally LGTM

**Comment by fejta** [comment]
I'll wait a bit until Sen either delegates to me or times out :)

**Comment by KesavanKing** [comment]
Ohh Ok.. My PR https://github.com/kubernetes/perf-tests/pull/187 which got merged  recently  and it is already in generalized way of calling perf-test via kubetest ```test-cmd```
For Example:
```kubetest --test-cmd  $GOPATH/src/k8s.io/perf-tests/run-e2e.sh --test-cmd-args --cluster-loader```
```kubetest --test-cmd  $GOPATH/src/k8s.io/perf-tests/run-e2e.sh --test-cmd-args --network-performance```


Thanks @krzyzacy for the info on ```test-cmd```.

**Pull request closed without merging**